images/win/scripts/Installers/Install-Ruby.ps1

```diff
@@ -0,0 +1,144 @@
+################################################################################
+##  File:  Install-Ruby.ps1
+##  Desc:  Install rubyinstaller2
+################################################################################
+
+function Get-RubyVersions
+{
+    param (
+        [System.String] $Arch = "x64",
+        [System.String] $Extension = "7z"
+    )
+
+    $uri = "https://api.github.com/repos/oneclick/rubyinstaller2/releases"
+    try
+    {
+        $versionLists = @{}
+        $assets = (Invoke-RestMethod -Uri $uri).Where{ -not $_.prerelease }.assets
+        $7zArchives = $assets.Where{ $_.name.EndsWith("$Arch.$Extension") }
+        $majorMinorGroups = $7zArchives | Group-Object { $_.name.Replace("rubyinstaller-", "").Substring(0, 3) }
+        foreach($majorMinorGroup in $majorMinorGroups)
+        {
+            $group = $majorMinorGroup.Group
+            $sortVersions = $group | Sort-Object {[Version]$_.name.Replace("rubyinstaller-", "").Replace("-$Arch.$Extension","").Replace("-",".")}
+            $latestVersion = $sortVersions | Select-Object -Last 1
+            $versionLists[$majorMinorGroup.Name] = $latestVersion.browser_download_url
+
+        }
+        return $versionLists
+    }
+    catch
+    {
+        Write-Host "Enable to send request to the '$uri'. Error: '$_'"
+        exit 1
+    }
+}
+
+# Most of this logic is from
+# https://github.com/MSP-Greg/actions-ruby/blob/master/lib/main.js
```

Comment:
This comment is a bit old and the URL doesn't work anymore, so I'd suggest linking to https://github.com/ruby/setup-ruby/blob/master/windows.js instead

Reply:
Thank you. Done.